ttnn/CMakeLists.txt

```diff
@@ -671,7 +671,7 @@ if(WITH_PYTHON_BINDINGS)
     list(
         APPEND
         TTNN_PUBLIC_LINK_LIBRARIES
-        pch_pybinds
+        pybind11::module
```

Comment:
Dumb question - but are these headers pre-compiled?

Reply:
Not sure if it matters too much on CI... on CI we have ccache so should make no huge difference

Locally it might matter. Although not sure how much it really makes a diff since the real changes are in our pybind files, rather than pybind itself. Pybind itself shouldn't be too large

Reply:
I believe we turn off pre-compiled headers when building with `ccache`

Reply:
👍 

What about local? Do we care

Reply:
wonder if we should shoehorn devs to using `ccache`. Does the unity setup work automatically?

Reply:
I use ccache locally and its nice :)

Reply:
unity setup does work automatically, and is on by default. ccache is not on by default. 

I say we FAFO, merge this PR and address pch if any complaints arise.

Reply:
FAFO